src/plone/app/multilingual/__init__.py

```diff
@@ -1,15 +1,15 @@
+from importlib.metadata import distribution
+from importlib.metadata import PackageNotFoundError
 from zope.i18nmessageid import MessageFactory
 
 import logging
-import pkg_resources
 
 
 try:
-    pkg_resources.get_distribution("Products.LinguaPlone")
-except pkg_resources.DistributionNotFound:
-    isLPinstalled = False
-else:
+    distribution("Products.LinguaPlone")
     isLPinstalled = True
+except PackageNotFoundError:
+    isLPinstalled = False
```

Comment:
LinguaPlone was Archetypes-based and therefore for Python 2 only. Can we also remove the code that checked for this?

Reply:
Done 👍🏾